client/src/components/categorical/index.tsx

```diff
@@ -27,19 +30,23 @@ class Categories extends React.Component<{}, State> {
     super(props);
     this.state = {
       createAnnoModeActive: false,
+      // eslint-disable-next-line react/no-unused-state --- FIXME: disabled temporarily
```

Comment:
These ESLint comments were needed to get ESLint to pass. I don't fully understand what this stuff is doing, but I do see some usages of `newCategoryText` and `categoryToDuplicate` within the repo so I don't think it's safe to remove them, even though ESLint thinks they're unused.